app/scripts/metamask-controller.js

```diff
@@ -133,7 +133,7 @@ module.exports = class MetamaskController extends EventEmitter {
     this.publicConfigStore = this.initPublicConfigStore()
 
     // manual disk state subscriptions
-    this.txController.store.subscribe((state) => {
+    this.txController.txStateManager.store.subscribe((state) => {
```

Comment:
lets actually subscribe to the `txController` store. if its doesnt have one, it should set the `txStateManager`'s store as its own

Reply:
👍